### DISCUSSION TITLE
Unable to revert to the original repository after listing issues.

### DISCUSSION THREAD

**Discussion by improve100** [magit/forge#576] [Support] [answered]
OS: Ubuntu
emacs: 27.3
emacs Release Version: doom emacs

I started in dired mode in the repository, entered magit, and then used `@ l i` to view the issue. I kept pressing `q` to exit, hoping to return to the original dired mode in the repository, but it didn't work in the end.
![Peek 2023-04-26 09-08](https://user-images.githubusercontent.com/6092914/234442775-0b283a4d-f853-42e0-a1bc-33b4bfdd4635.gif)

**Comment by tarsius** [comment, marked as answer]
I get the behavior that you want.


It appears that quitting the `tabulated-list-mode` derived buffer does not return to the previous buffer, not quitting from a `magit-mode` derived buffer. So I would look into customization for that first, only then Magit.

But then also reset all options in the `magit-buffers` custom group to their defaults.
